schema_tools/csv_to_scheme.py

```diff
@@ -132,6 +132,22 @@ def generate_scheme(header_name, codes):
       "NumericValue": -110,
       "StringValue": "CE",
       "VisibleInCoda": False
+    },
+    {
+      "CodeID" : "code-PB-a434a800",
+      "DisplayText"    : "Push back",
```

Comment:
Normal codes usually have lower case display text

Reply:
Not sure if you want to do the same thing here

Reply:
Done, thanks